binding/dotnet/PorcupineTest/MainTest.cs

```diff
@@ -90,54 +171,137 @@ public void TestProcessMultiple()
                 BuiltInKeyword.PORCUPINE,
                 BuiltInKeyword.TERMINATOR
             };
+            porcupine = Porcupine.FromBuiltInKeywords(ACCESS_KEY, inputKeywords);
+            runProcess(
+                "multiple_keywords.wav",
+                new List<int>() {7, 0, 1, 2, 3, 4, 5, 6, 7, 8});
+        }
 
-            List<BuiltInKeyword> expectedResults = new List<BuiltInKeyword>() {
-                BuiltInKeyword.PORCUPINE,
-                BuiltInKeyword.ALEXA,
-                BuiltInKeyword.AMERICANO,
-                BuiltInKeyword.BLUEBERRY,
-                BuiltInKeyword.BUMBLEBEE,
-                BuiltInKeyword.GRAPEFRUIT,
-                BuiltInKeyword.GRASSHOPPER,
-                BuiltInKeyword.PICOVOICE,
-                BuiltInKeyword.PORCUPINE,
-                BuiltInKeyword.TERMINATOR
+        [TestMethod]
+        public void TestSingleKeywordDe()
+        {
+            string language = "de";
+            List<string> keywords = new List<string>() {"heuschrecke"};
+
+            porcupine = Porcupine.FromKeywordPaths(
+                ACCESS_KEY,
+                getKeywordPaths(language, keywords),
+                getModelPath(language),
+                Enumerable.Repeat(0.5f, keywords.Count())
```

Comment:
the binding does this by default, so the only reason to pass in values is if we want to use custom (i.e. not 0.5) values